### ISSUE TITLE
Does not seem to build neatly with stack.

### ISSUE THREAD

**Issue by pharaun** [jakubfijalkowski/hlibsass#2]
```
[paul@imaskar:~/projects/programming/alldice] (master) :) $ stack build                                                                                 12:49AM
alldice-0.1.0.0: configure
Configuring alldice-0.1.0.0...
alldice-0.1.0.0: build
Preprocessing executable 'alldice' for alldice-0.1.0.0...
[8 of 8] Compiling Main             ( src/Main.hs, .stack-work/dist/x86_64-linux/Cabal-1.22.4.0/build/alldice/alldice-tmp/Main.o )

src/Main.hs:4:1: Warning:
    The import of ‘Control.Applicative’ is redundant
      except perhaps to import instances from ‘Control.Applicative’
    To import instances alone, use: import Control.Applicative()
Linking .stack-work/dist/x86_64-linux/Cabal-1.22.4.0/build/alldice/alldice ...
/usr/lib/gcc/x86_64-pc-linux-gnu/4.9.3/../../../../x86_64-pc-linux-gnu/bin/ld: cannot find -lsass
collect2: error: ld returned 1 exit status

--  While building package alldice-0.1.0.0 using:
      /home/paul/.stack/setup-exe-cache/setup-Simple-Cabal-1.22.4.0-x86_64-linux-ghc-7.10.2 --builddir=.stack-work/dist/x86_64-linux/Cabal-1.22.4.0/ build exe:alldice --ghc-options  -ddump-hi -ddump-to-file
    Process exited with code: ExitFailure 1
```

Basically this is what happens when I try to use this project with stack, it falls apart. I am not for certain the exact reasoning and I'll be linking this ticket to the stack project in case it has to do with the submodule here (libsass). But if I alter my stack config to include:

```
# Local packages, usually specified by relative directory name
packages:
- '.'
- location: libs/hlibsass
  extra-dep: true
```

Then make this repo a submodule of my regular repo, and force the libsass submodule to load, stack is able to build and relink against the project over and over just fine.

**Comment by jakubfijalkowski** [comment]
I think that this is a problem with stack or at least with my lack of knowledge of it. Cabal-install invokes "copy" when installing dependencies - stack does not, this in turn makes `libsass.a` (that has been built during configure phase) disappear after installation (was in `tmp`) of the `hlibsass`. That's why the first build succeeds and next ones fail.

There is another workaround available - build `libsass` separately and set correct `extra-lib-dirs` in `stack.yaml`. You can also build `hlibsass` with `externalLibsass` flag set.

**Comment by jakubfijalkowski** [comment]
Fixed in stack - commercialhaskell/stack#1203 and commercialhaskell/stack@f71a515 .
